test/ExportQueryExecutionTreesTest.cpp

```diff
@@ -1125,8 +1130,7 @@ TEST(ExportQueryExecutionTrees, getIdTablesReturnsSingletonIterator) {
   Result result{idTable.clone(), {}, LocalVocab{}};
   auto generator = ExportQueryExecutionTrees::getIdTables(result);
 
-  EXPECT_THAT(convertToVector(std::move(generator)),
-              ElementsAre(Eq(std::cref(idTable))));
+  EXPECT_THAT(convertToVector(std::move(generator)), matchIdTables(idTable));
```

Comment:
I could also factor this out to a helper function, but the necessary `source_location`  makes the implementation a bit ugly at least.

But maybe I'll think about whether this is worth it
(maybe I can even put the `getIdTables` function into the common logic.